dev-server/templates/index.mustache

```diff
@@ -21,6 +21,7 @@
     <li><a href="/document/tolstoy">Anna Karenina, by Leo Tolstoy</a></li>
     <li><a href="/document/z-index">z-index playground</a></li>
     <li><a href="/document/shadow-dom">shadow DOM playground</a></li>
+    <li><a href="/document/multi-frames">multi-frames</a></li>
```

Comment:
Nitpick: Can you capitalize the document titles here ("Shadow DOM test", "Multi-frame test" etc.)

Reply:
Done